src/CMakeLists.txt

```diff
@@ -19,7 +19,9 @@ get_filename_component(
     MXNET_PROJECT_SOURCE_DIR
     "${CMAKE_CURRENT_SOURCE_DIR}/../../.."
     ABSOLUTE)
-include("${MXNET_PROJECT_SOURCE_DIR}/cmake/ChooseBlas.cmake")
+if(APPLE)
+    include("${MXNET_PROJECT_SOURCE_DIR}/cmake/ChooseBlas.cmake")
+endif()
```

Comment:
Could you give a little context for this hunk?

Reply:
This was in PR #8. It causes blas problems on CentOS. cmake complains about not finding Atlas, even though Atlas is already installed and `USE_BLAS` is not used, e.g. `make -j$(nproc) USE_NGRAPH=1` fails to build due to Atlas problems. Is ChooseBlas needed for OS X?

Reply:
> Is ChooseBlas needed for OS X?

I'm not sure, but I would suggest testing this on OS X before merging the PR.

If you decide to keep this hunk, it might be nice to add a small comment explaining what you told me.

Reply:
Here's my concern.  There are two bodies of code that use the mshadow headers: 
- The mxnet-ngraph bridge code
- The rest of mxnet

The reason we include `ChoseBlas.cmake` here is to ensure that when both of those bodies of code are compiled, the same preprocessor flags are in effect during the compilation of the mshadow headers.  It's not clear what the consequences of a mismatch would be on Linux, but IIRC on OSX it was problematic.

So all things considered, I'd be more comfortable figuring out:
- what's going wrong with this logic on CentOS, and
- why the rest of MXnet isn't broken on CentOS due to `ChoseBlas.cmake`.

That being said, there's an outstanding upstream PR for refactoring / rationalizing how MXnet's CMake build system deals with math libraries.  So if this passes testing I think we can live with it as an interim solution.